src/uController.SourceGenerator/MinimalCodeGenerator.cs

```diff
@@ -383,8 +435,8 @@ private void EmitParameter(ref bool hasAwait, ref bool hasFromBody, ref bool has
                 }
                 else
                 {
-
-                    WriteLine($"var {parameterName} = await ResolveBody<{parameter.ParameterType}>(httpContext);");
+                    WriteLine(
+                              $"var {parameterName} = await ResolveBody<{parameter.ParameterType}>(httpContext, {(isOptional ? "true" : "false")});");
```

Comment:
Why this formatting change?

```suggestion
                    WriteLine($"var {parameterName} = await ResolveBody<{parameter.ParameterType}>(httpContext, {(isOptional ? "true" : "false")});");
```